src/core/io/ibprof_plain.c

```diff
@@ -168,13 +169,13 @@ static void _ibprof_module_dump(FILE* file, IBPROF_MODULE_OBJECT *module_obj, IB
 	case IBPROF_MODE_ERR:
 		plain_output(file, "%-30.30s : %10s   %10s   %10s   %10s   %10s   %10s\n",
 			(module_obj->name ? module_obj->name : "unknown"), "count",
-						"total(ms)", "avg(ms)", "max(ms)", "min(ms)", "fail");
+						"total", "avg", "max", "min", "fail");
```

Comment:
@liranoz12 it would be nice to keep such information. Please consider adding short name as (`'s '`, `'ms'`, `'us'`). I remember that there was a request to provide one as more informative.